pkg/upgrade/upgrades/wait_for_del_range_in_gc_job.go

```diff
@@ -83,7 +83,10 @@ func waitForDelRangeInGCJob(
          elements AS (SELECT * FROM tables UNION ALL SELECT * FROM indexes)
   SELECT id
     FROM elements
-   WHERE COALESCE(progress->>'status' NOT IN ('WAITING_FOR_MVCC_GC', 'CLEARED'), true)
+	-- While we are waiting for the GC TTL WAITING_FOR_CLEAR is equivalent to NULL,
```

Comment:
nit: should WAITING_FOR_CLEAR say WAITING_FOR_MVCC_GC?

Reply:
So, what's happening here is that WAITING_FOR_CLEAR=0, and `json:"status,omitempty"` so it will not exist. Let me update the comment for more clarity.

-- While we are waiting for the GC TTL, the status will be WAITING_FOR_CLEAR because omitempty
-- set on this field it will not exist in the JSON output. Because tombstone adoption unconditionally 
-- enabled by an earlier version, we should be safe to skip any job that hasn't started GCing yet.